src/chat/fetchingReducer.js

```diff
@@ -51,6 +56,7 @@ export default (state: FetchingState = initialState, action: Action): FetchingSt
       return messageFetchStart(state, action);
 
     case MESSAGE_FETCH_COMPLETE:
+      // We don't want to accumulate old searches that we'll never need again.
```

Comment:
Cool, looks like the `isSearchNarrow` logic was moved so it happens inside `messageFetchComplete`, following https://github.com/zulip/zulip-mobile/pull/3868#discussion_r411815301. It seems to me that this comment should only appear once (not twice), inside `messageFetchComplete`, right above the code it explains the reason for.

Reply:
Oops, I copied this line above, but forgot to delete it here.